locales/vi-VN/App.json

```diff
@@ -243,5 +238,7 @@
   "Label": "Nhãn",
   "Last used": "Lần cuối cập nhật",
   "Transactions": "Giao dịch",
-  "ALPH amount": "Số lượng ALPH"
+  "ALPH amount": "Số lượng ALPH",
+  "Set lock time": "Set lock time",
+  "Tweak gas settings": "Tweak gas settings"
```

Comment:
@nit27 I could use your help with these 2 new sentences! You can just comment here and I'll add it. 🙏 Much obliged!

Reply:
"Set lock time": "Đặt thời gian khoá",
 "Tweak gas settings": "Tuỳ chỉnh cài đặt gas"
✈️

Reply:
"Set lock time": "Đặt thời gian khoá",
  "Tweak gas settings": "Tuỳ chỉnh cài đặt gas"